projects/angular/components/ui-grid/src/ui-grid.component.html

```diff
@@ -47,7 +47,8 @@
         </ng-container>
     </div>
 
-    <div *ngIf="selectionManager.hasValue()"
+    <div #gridActionButtons
+         *ngIf="hasSelection$ | async "
```

Comment:
There are other places where we can switch to this stream, could we update all the places where `selectionManager.hasValue()` is being used?
```suggestion
         *ngIf="hasSelection$ | async"
```